rewrite-maven/src/test/java/org/openrewrite/maven/AddMavenWrapperChecksumValidationTest.java

```diff
@@ -0,0 +1,24 @@
+package org.openrewrite.maven;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.properties.Assertions.properties;
+
+public class AddMavenWrapperChecksumValidationTest implements RewriteTest {
```

Comment:
Could you add another test to verify that we do not make any changes if there is already a checksum?

Reply:
Sure, there is more to do, also a maven-wrapper.jar sha256 verification.
Just wanted to use the opportunity of EAP review to determine whether the rigth approach was chosen.

Reply:
Done